packages/bits/spec/components/form-field/form-field.e2e.ts

```diff
@@ -227,21 +196,20 @@ describe("USERCONTROL form-field >", () => {
             await timepicker.icon.getElement().click();
             expect(await timepicker.textbox.input.getId())
                 .toEqual(await (await browser.switchTo().activeElement()).getId());
-
             await timepicker.icon.getElement().click();
         });
 
-        it("should change model for dateTimePicker", async () => {
```

Comment:
Does this old test work as it was previously written? If so, it might be good to restore and enable the old version and create a new disabled test that recreates the bug you found.

Reply:
Blake, no, the test was false negative, so it doesn't work correctly and was shown as passed every time, though the error was there. That's why we didn't found the bug before.
So I would like to rewrite the test to cover both situations.